app/src/main/java/com/kickstarter/libs/rx/transformers/Transformers.java

```diff
@@ -184,4 +185,11 @@ private Transformers() {}
   public static @NonNull <T> ObserveForUITransformer<T> observeForUI() {
     return new ObserveForUITransformer<>();
   }
+
+  /**
+   * Emits the second of any input pair
+   */
+  public static <R> R second(final @NonNull Pair<?, R> anyPair) {
```

Comment:
just to clarify, we want this in pair utils cause it's a nice lil pure function that we can use in any context, not just with observables. the `Transformers.java` file is about creating new operators on observables.